inc/Ui/Admin.php

```diff
@@ -0,0 +1,172 @@
+<?php
+namespace dokuwiki\Ui;
+
+/**
+ * Class Admin
+ *
+ * Displays the Admin screen
+ *
+ * @package dokuwiki\Ui
+ * @author Andreas Gohr <andi@splitbrain.org>
+ * @author Håkan Sandell <hakan.sandell@home.se>
+ */
+class Admin extends Ui {
+
+    protected $menu;
+
+    /**
+     * Display the UI element
+     *
+     * @return void
+     */
+    public function show() {
+        $this->menu = $this->getPluginList();
+        echo '<div class="UI-admin">';
```

Comment:
We don't use CSS classes with uppercase letters anywhere else. I find it can create confusion and be more error-prone and there might be problems with some older browsers.